MITADS/opensubtitles_exporter.py

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/env python3
+
+from typing import Pattern
+import re
+from pathlib import Path
+from unidecode import unidecode  
+import time
+
+
+mapping_normalization = [
+
+  ##L'Ordine delle entry (replace e sub con regex) è significativo!!!
+
+  ##se la frase contiene parole con underscore o  @, allora non fa parte di una conversazione
+  ##[ re.compile('(^|(.*\s))\w+(_|@|®|\+|\{|\})\w+((\s.*)|$)'), u'' ],
+
+  ##se la frase inizia con un numero , viene eliminata
+  [ re.compile('^\d+(.*)'), u'' ],  
+
+  ##tutto ciò è contenuto tra paretesi va eliminato, non fa parte di una conversazione    
+  [ re.compile('(\(|\[|{)[^(\)|\]|})]*(\)|\]|})'), u'' ],
+
+  ## potrebbero essere parentesi annidate(ci sono nel dataset), lo lancio due volte
```

Comment:
Can we do a better regex that remove parenthesis and avoid another regex that slow down the script?

Reply:
I have not succeeded. I launch the same regex  for nested parentheses